<?xml version="1.0"?>

<launch>
    <!-- FSM -->
    <node pkg="solamr" type="fsm.py" name="fsm">
        <param name="file_path_single_amr" value="$(find solamr)/launch/run_single_AMR.launch"/>
        <param name="file_path_single_assembled" value="$(find solamr)/launch/run_single_assembled.launch"/>
    </node>
    <node pkg="smach_viewer" type="smach_viewer.py" name="smach_viewer"/>
    <node pkg="rqt_console" type="rqt_console" name="rqt_console"/>
</launch>
